test/system/710-kube.bats

```diff
@@ -135,4 +135,44 @@ status  | =  | null
     run_podman rmi $(pause_image)
 }
 
+@test "podman kube generate - deployment" {
+    local pname=p$(random_string 15)
+    local cname1=c1$(random_string 15)
+    local cname2=c2$(random_string 15)
+
+    run_podman pod create --name $pname
+    run_podman container create --name $cname1 --pod $pname $IMAGE top
+    run_podman container create --name $cname2 --pod $pname $IMAGE bottom
+
+    skip_if_remote "containersconf needs to be set on server side"
+    containersconf=$PODMAN_TMPDIR/containers.conf
+    cat >$containersconf <<EOF
+[engine]
+kube_generate_type="deployment"
+EOF
+    CONTAINERS_CONF=$containersconf run_podman kube generate $pname
```

Comment:
Please use `CONTAINERS_CONF_OVERRIDE`. The older form is not spoken of or used if at all possible.

Reply:
fixed